frontend/app-dashboard/js/view/panel-dashboard.js

```diff
@@ -157,7 +157,17 @@ define([
                 let title = this.sub_region_title_template;
                 let item_template = this.sub_region_item_template;
                 let $table = $('<table></table>');
-                let pivot_data = this.panel_handlers_hash['population'].stats_data
+                let pivot_data = this.panel_handlers_hash['population'].stats_data;
+                if (!pivot_data[0]) {
+                    $.each(this.panel_handlers_hash, function( key, value ) {
+                        if(value.stats_data[0]) {
+                            pivot_data = value.stats_data
+                        }
+                    });
+                }
+                if(pivot_data[0]) {
+                    return
```

Comment:
Why return?
It will skip the rests of the code?

Reply:
i'm sorry, forgot to remove this